.gitignore

```diff
@@ -66,3 +66,5 @@ zipline.iml
 
 # PyCharm custom settings
 .idea
+
+TAGS
```

Comment:
does this just go in your home dir? I will add this here but I don't see harm in putting it here also